systemd does not include pkg-config dependencies for static linking.

diff -Naur a/src/libsystemd/libsystemd.pc.in b/src/libsystemd/libsystemd.pc.in
--- a/src/libsystemd/libsystemd.pc.in	2022-01-18 11:35:43.000000000 +0000
+++ b/src/libsystemd/libsystemd.pc.in	2022-03-05 21:24:47.139338880 +0000
@@ -18,3 +18,4 @@
 Version: {{PROJECT_VERSION}}
 Libs: -L${libdir} -lsystemd
 Cflags: -I${includedir}
+Requires: libcap
